rpg/rpg.js

```diff
@@ -193,7 +193,7 @@ function BioString(userID){
 }
 
 // Events
-function RunEvent(userID, channelID){
+function RunEvent(userID, channelID, goToPage=false){
```

Comment:
Can we set the default of goToPage to be 0?
It'll still return false for our if statements and we don't have to expect it to be a bool or int